### PR TITLE
Add docker and postgres configuration

### PR DESCRIPTION
**Pull request by rafalciecwierzmoodup**
**Acceptance criteria**
Our backend is able to communicate with a PostgreSQL database that we have running in docker container

To demonstrate that the database works, / endpoint ("hello world" endpoint) returns a list of users' emails fetched from the database's users table

**Technical notes**
We will use Prisma ORM for schema management and migrations - [Prisma | NestJS - A progressive Node.js framework](https://docs.nestjs.com/recipes/prisma)

Create a service that will perform simple database calls

Consider also using a repository pattern

Use Prisma migration (raw SQL) to create a users  table and populate it with few users (name and email will be enough)

### CODE REVIEW

File: .example.env  [hunk 1]
```diff
@@ -0,0 +1,4 @@
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+DATABASE_URL="postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:5555/${POSTGRES_DB}?schema=public"
```
Comment (rad20c) [inline .example.env :4]: hmm, why custom port (5555)?

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+save-exact=true
```
Comment (rad20c) [inline .npmrc:1]: 👍🏻

File: src/users/users.controller.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  @Patch(':id')
+  update(@Param('id') id: string) {
+    return this.usersService.update(id);
+  }
+
+  @Delete(':id')
+  remove(@Param('id') id: string) {
+    return this.usersService.remove(id);
+  }
```
Comment (rad20c) [inline src/users/users.controller.ts]: I know that this is how nest generates CRUD, but I would add return types for all of these (same for service).

Right now for `findAll` you are returning `User` type that comes from `import { User } from '@prisma/client';`. It's better to separate database model / entity from presentation model. It's not a blocker for this PR (can be done in the next one), but I would add a `dto` folder where I would define requests and response DTO classes (here we need just a response DTO class). This way we have more control over what data we are returning from what endpoint.

Also note that there are mixed opinions whether to define DTOs that are shared between controllers / routes or define separate DTO for each route. I'm personally a fan of having a separate DTO for each route. Although it's more duplicated code this way, we can later freely change these DTOs separately without affecting others.

File: src/users/users.service.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    return `This action returns a #${id} user`;
+  }
+
+  update(id: string) {
+    return `This action updates a #${id} user`;
+  }
+
+  remove(id: string) {
+    return `This action removes a #${id} user`;
+  }
```
Comment (rad20c) [inline src/users/users.service.ts]: I'd add return types

**Review by rad20c — APPROVED** [review]

File: .example.env  (hunk 1 above)
Comment (rafalciecwierzmoodup) [reply .example.env :4]: I have another postgress container running with strapi and I don't want to kill it :p

**Pull request merged**